rpcs3/Emu/Cell/Modules/cellMsgDialog.cpp

```diff
@@ -124,6 +119,14 @@ s32 cellMsgDialogOpen2(u32 type, vm::cptr<char> msgString, vm::ptr<CellMsgDialog
 	return CELL_OK;
 }
 
+s32 cellMsgDialogOpen(u32 type, vm::cptr<char> msgString, vm::ptr<CellMsgDialogCallback> callback, vm::ptr<void> userData, vm::ptr<void> extParam)
+{
+	//Note: This function needs proper implementation, solve first argument "type" conflict with MsgDialogOpen2 in cellMsgDialog.h.
+	LOG_TODO(HLE, "cellMsgDialogOpen Unimplemented using cellMsgDialogOpen2 instead");
```

Comment:
I'd like it use cellSysutil.todo() formatting style, print arguments and propagate return value.